src/test/java/org/folio/rest/api/IsbnNormalizationTest.java

```diff
@@ -0,0 +1,166 @@
+package org.folio.rest.api;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.folio.rest.api.StorageTestSuite.TENANT_ID;
+import static org.folio.rest.api.StorageTestSuite.storageUrl;
+import static org.folio.util.StringUtil.urlEncode;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.rest.support.ApiTests;
+import org.folio.rest.support.Response;
+import org.folio.rest.support.builders.RequestItemSummary;
+import org.folio.rest.support.builders.RequestRequestBuilder;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.HttpResponse;
+import lombok.SneakyThrows;
+
+public class IsbnNormalizationTest extends ApiTests {
+  private static final String REQUEST_STORAGE_URL = "/request-storage/requests";
+
+  @SneakyThrows
+  @Before
+  public void beforeEach() {
+    StorageTestSuite.deleteAll(storageUrl(REQUEST_STORAGE_URL));
+  }
+
+  @SneakyThrows
+  @Test
+  public void searchForNormalizedIsbns() {
+    createRequests();
+
+    //canSearchForFirstIsbnWithAdditionalHyphens()
+    find("isbn = 0-552-16754-1",      "Interesting Times");
+    //canSearchForFirstIsbnWithAdditionalHyphenAndTruncation()
+    find("isbn = 05-5*",              "Interesting Times");
+    //canSearchForSecondIsbnWithMissingHyphens()
+    find("isbn = 9780552167543",      "Interesting Times");
+    //canSearchForSecondIsbnWithMissingHyphensAndTrunation()
+    find("isbn = 9780*", "Interesting Times", "Temeraire");  
+    //canSearchForSecondIsbnWithAlteredHyphens()
+    find("isbn = 9-7-8-055-2167-543", "Interesting Times");
+    //cannotFindIsbnWithTailString()
+    find("isbn = 552-16754-3");
+    //cannotFindIsbnWithInnerStringAndTruncation()
+    find("isbn = 552*");
+  }
```

Comment:
Now that the test request creation runs consistently during these tests, please can these tests be split up into separate examples again.

During each test, please only create the items that are needed to demonstrate that example.